CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [5.2.0](https://github.com/theforeman/puppet-dns/tree/5.2.0) (2018-05-09)
+
+[Full Changelog](https://github.com/theforeman/puppet-dns/compare/5.1.0...5.2.0)
+
+**Implemented enhancements:**
+
+- Adds control keys and specifying update policy [\#108](https://github.com/theforeman/puppet-dns/pull/108) ([zyronix](https://github.com/zyronix))
+
+**Closed issues:**
+
+- Fails with puppetlabs/concat 4.1.1 [\#107](https://github.com/theforeman/puppet-dns/issues/107)
+- Add support for adding keys for nsupdate [\#94](https://github.com/theforeman/puppet-dns/issues/94)
+
+**Merged pull requests:**
+
+- Pin rdoc on ruby 2.1 [\#106](https://github.com/theforeman/puppet-dns/pull/106) ([ekohl](https://github.com/ekohl))
+- modulesync: signed tags [\#105](https://github.com/theforeman/puppet-dns/pull/105) ([ekohl](https://github.com/ekohl))
```

Comment:
This one has the label modulesync. Normally we skip those since they're not relevant for end users. I've used `--exclude-labels duplicate,question,invalid,wontfix,Modulesync` for this and we probably need to include it in the documentation.